src/registrar/context_processors.py

```diff
@@ -108,10 +108,20 @@ def is_widescreen_mode(request):
         "/no-organization-domains/",
         "/domain-request/",
     ]
+    exclude_paths = [
```

Comment:
widescreen_paths can be a bear as it trickles down sub-urls. exclude_paths gives us a way out.